src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -1788,31 +1802,31 @@ void ViewSceneDialog::setAmbientValue(double value)
 {
   state_->setValue(Modules::Render::ViewScene::Ambient, value);
   setMaterialFactor(SRInterface::MAT_AMBIENT, value);
-  newGeometryValue();
+  newGeometryValue(true);
```

Comment:
Thus this type of call would be `forceNewGeometry` or `newGeometryValueWithCleanup` or whatever